### PR TITLE
Better interface for transifex

### PR DESCRIPTION
**Pull request by mkangia**
Just want to help the user confirm what they want to achieve and also add a confirmation step.
Open to suggestions to do this better and if and how can we translate the broken message for the current user.

buddy @dannyroberts

### CODE REVIEW

**Comment by orangejenny** [comment]
Hi Manish, could you summarize the changes and add some screenshots or a link to staging?@ohmegasquared might have suggestions - we were just talking a couple weeks back about when to use popups and how to have users confirm dangerous actions. (I'm not a fan of the default browser confirmation - it's jarring and people don't necessarily read it.)

**Comment by mkangia** [comment]
sure will do,
need to rethink confirmation message so closing this out temporarily so that you dont get notifications till then.

**Comment by ohmegasquared** [comment]
Happy to help you rethink messaging!
